frontend/package-lock.json

```diff
@@ -5,22 +5,17 @@
   "requires": true,
```

Comment:
Can you remove this entire file from your diff? If you didn't add any new package, don't change this file.

Reply:
Removed.